docs/gettingstarted/quickstart.rst

```diff
@@ -0,0 +1,313 @@
+Getting started with Flocker
+============================
+
+.. logo::
+
+.. intro-text::
+
+   Flocker lets you run microservices apps with database containers and move them around between servers. It comes in two pieces and you’ll need both.
+
+.. contents::
+   :local:
+
+.. tutorial-step::
+
+   Step 1: Installing Flocker CLI & Node
+   =====================================
+
+   .. parallel::
+
+      .. mobile-label::
+
+         Local
+
+      .. image:: images/macbook.png
+         :class: center-block img-responsive
+         :alt: Flocker CLI diagram
+
+      Flocker CLI
+      -----------
+
+      +--------------------------------------------------------------------------+
+      | Runs on your laptop                                                      |
+      +--------------------------------------------------------------------------+
+      | Uses application and deployment configuration files                      |
+      +--------------------------------------------------------------------------+
+      | Deploys containers to a cluster of servers                               |
+      +--------------------------------------------------------------------------+
+
+      .. empty-div:: arrow-down center-block hidden-xs hidden-sm
+
+   .. parallel::
+
+      .. mobile-label::
+
+         Live
+
+      .. image:: images/nodes.png
+         :class: center-block offset-top img-responsive
+         :alt: Flocker Node diagram
+
+      Flocker Node
+      ------------
+
+      +--------------------------------------------------------------------------+
+      | Runs on each server in a cluster                                         |
+      +--------------------------------------------------------------------------+
+      | Links, ports and volumes work across hosts                               |
+      +--------------------------------------------------------------------------+
+      | After deployment, containers can move around                             |
+      +--------------------------------------------------------------------------+
+
+      .. empty-div:: arrow-down center-block
+
+   .. parallel::
+
+      Installation
+      ------------
+
+      .. mobile-label::
+
+         Local
+
+      .. noscript-content::
+      
+         OS X
+         ^^^^
+
+         Install the flocker-cli client on your Mac (requires Homebrew):
+
+         .. code-block:: console
+
+            $ brew update && \
+              brew tap clusterhq/flocker && \
+              brew install flocker-0.3.2
+            
+      .. noscript-content::
+
+         Ubuntu / Debian
+         ^^^^^^^^^^^^^^^
+
+         Install the dependencies:
+         
+         .. code-block:: console
+         
+            $ sudo apt-get install gcc python2.7 python-virtualenv python2.7-dev
+
+         Next, install the flocker-cli client on your Linux machine:
+
+         .. code-block:: console
+
+            $ virtualenv flocker-tutorial && \
```

Comment:
I copy/pasted this command and the `$` was copied.  I think there's an easy fix for this, something to do with `.. prompt:: bash $`.  If that fixes it and still looks okay, please make that change.  If it's any harder than that, file a follow-up issue instead and leave this as-is for now.